state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -241,11 +253,13 @@ pub mod pallet {
 
 	/// Scheduled Swaps
 	#[pallet::storage]
+	#[pallet::getter(fn swap_queue)]
 	pub(crate) type SwapQueue<T: Config> =
 		StorageMap<_, Twox64Concat, BlockNumberFor<T>, Vec<Swap>, ValueQuery>;
 
 	/// The first block for which swaps haven't yet been processed
 	#[pallet::storage]
+	#[pallet::getter(fn first_unprocessed_block)]
```

Comment:
I'm not sure there is much point in using these getters really, particularly as you cannot update through a similarly named setter?

Reply:
I would say it is cleaner than writing `pallet_cf_swapping::FirstUnprocessedBlock::<Runtime>::get()`. I also don't think every getter needs a setter (in fact, it is a common incapsulation pattern to expose a value through a getter only so that it can only be modified internally).

Reply:
"(in fact, it is a common incapsulation pattern to expose a value through a getter only so that it can only be modified internally)", yeah but that isn't hwat you are doing here.

"IMO it is a bit cleaner than writing pallet_cf_swapping::FirstUnprocessedBlock::<Runtime>::get()" I disagree as naming the storage item you are reading is inherently clearer than using some other name for it (When that other name isn't consistently used elsewhere), and I believe there are basically zero places we use these getters substrate generates elsewhere in the codebase.

Reply:
If we do what I suggest inregards to the code that uses this getter, we wouldn't even need to read this storage item anyway.